pyconcepticon/commands.py

```diff
@@ -417,3 +417,56 @@ def lookup(args):
         for f in found:
             writer.writerow(f)
         print(writer.read().decode('utf-8'))
+
+
+def check(args):
+    """
+    Identifies some issues with wordlists 
```

Comment:
I'd prefer "concept list" here, since this is the nomenclature in Conepticon. Wordlists otoh are the things lexibank deals with.